src/components/main/body/Menu.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import Sidebar from 'react-sidebar';
+
+// const buildDataUrl = () => {
+//   return `https://data.lacity.org/resource/${dataResources[year]}.json?$select=location,zipcode,address,requesttype,status,ncname,streetname,housenumber&$where=date_extract_m(CreatedDate)+between+${startMonth}+and+${endMonth}+and+requesttype='${request}'`;
+// };
+
+const Menu = () => {
+  const [isSidebarOpen, setIsSidebarOpen] = useState(true);
+  return (
+    <>
+      <Sidebar
+        open={isSidebarOpen}
+        sidebar={(
+          <div>
+            THIS IS A TEST
+            <button
+              type="button"
+              className="button is-primary"
+              onClick={() => setIsSidebarOpen(!isSidebarOpen)}
+            >
+              Show quickview
+            </button>
+          </div>
+        )}
+        styles={{ sidebar: { background: 'white' } }}
```

Comment:
Thoughts on stripping out the inline style and incorporating into Bulma styles so all styles are centralized? I can get the basic SASS stuff set up this week so we can start matching styles to the mockups.

Alternatively, it might be worthwhile to leave as-is until MVP is achieved.

https://bulma.io/documentation/customize/with-node-sass/